src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -747,7 +747,15 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
                     | CoreResponse.InfoRes msg | CoreResponse.ErrorRes msg ->
                         LspResult.internalError msg
                     | CoreResponse.Res(tip, signature, footer, typeDoc) ->
-                        match TipFormatter.formatTipEnhanced tip signature footer typeDoc with
+                        let formatCommentStyle =
+                            if config.TooltipMode = "full" then
+                                TipFormatter.FormatCommentStyle.FullEnhanced
+                            else if config.TooltipMode = "summary" then
+                                TipFormatter.FormatCommentStyle.SummaryOnly
+                            else
+                                TipFormatter.FormatCommentStyle.Legacy
```

Comment:
Why do we even need `Legacy` mode? You're setting `full` as default anyway in the config object, so this will never happen?

Reply:
I have no idea :p 

I remember that someone mentioned to keep Legacy for others tools like Vim/Emacs mode, etc. 

I was thinking people could set the legacy mode if they wanted, we could set legacy mode by default if we prefer. Ionide will always use either `full` or `summary` as per configuration in the `package.json` of Ionide.